requests_ntlm/requests_ntlm.py

```diff
@@ -55,11 +55,19 @@ def retry_using_http_NTLM_auth(self, auth_header_field, auth_header,
         request = response.request.copy()
 
         # initial auth header with username. will result in challenge
-        msg = "%s\\%s" % (self.domain, self.username) if self.domain else self.username
+        if not self.domain:
+            msg = self.username
```

Comment:
Can we please cast this? (`str(self.username)`, if you're not sure what I mean)
The cast is implicit in the string formatting below, but if we don't cast it there's nominally a slight behavioural change.